.history/app/css/app_20180305105330.css

```diff
@@ -0,0 +1,156 @@
+@font-face {
```

Comment:
you should add these files in gitignore.

Reply:
okay